trillian/integration/ct_functions.sh

```diff
@@ -8,6 +8,7 @@ CT_LIFECYCLE_CFG=
 CT_COMBINED_CFG=
 PROMETHEUS_CFGDIR=
 readonly CT_GO_PATH=$(go list -f '{{.Dir}}' github.com/google/certificate-transparency-go)
+readonly MONO_PATH="$(go list -f '{{.Dir}}' github.com/google/monologue/incident)/.."
```

Comment:
Do you have to use ".." because monologue has no top-level "monologue" package?

Reply:
Yep.